server/index.js

```diff
@@ -28,6 +34,14 @@ app.set('port', port);
 
 app.listen(port, () => {
   console.log(`App listening on port ${app.get('port')}`);
+
+  // add worker if on production environment
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
Is it necessary to use the if condition here? This would make the implementation not to be testable nor work in development without modification. You might consider removing the `if (process.env.NODE_ENV === 'production')`

Reply:
I dont think this is a good practice to do.
1. The worker should be able to run in development and test so that a developer can change the interval and see it work as expected, confirm before pushing changes to production.
2. When a developer exists the application, it should stop running.
3. Writing test will also never get inside that block because within test environment, `process.env.NODE_ENV` is 'test` and not `production`
What do you think @IAMOTZ

Reply:
Given that the worker has the potential to execute a lot of automation once it is triggered, one has to be very deliberate when triggering the worker, consequently, I would go with the idea of adding the environment(production) check.
If the worker is needed to be executed during development, it is as simple as removing the environment check.

Reply:
Yes it does, but that is after the usual 24 hours interval has reached, long enough for the developer's server to have restarted at least once except `TIMER_INTERVAL` was deliberately changed to 5 minutes by the developer for testing purpose. Also even if it runs services, they will only execute using development environment variables, whether slack or freckle or email, hence constitutes no risk to production. I stand to be corrected